client/src/pages/SearchResult/SingleSearchResult.tsx

```diff
@@ -47,6 +54,8 @@ export function SingleSearchResultPage() {
       try {
         const data = await getRestaurant(place_id);
         setRestaurantData(data.result);
+        setPrice(data.result.price_level);
```

Comment:
We don't need line 57 anymore since its being set on line 73 from the restaurantData object